ftva_lab_data/templates/partials/search_results_table.html

```diff
@@ -1,4 +1,4 @@
-{% include "partials/pagination_controls.html" %}
+{% include "partials/pagination_controls.html" with show_auxiliary_controls=True %}
```

Comment:
It took a few looks for me to understand that this applies only to the top row of nav controls.  I think this line is OK, but the second call (line 82) to show the bottom nav controls would be better with an explicit `with show_auxiliary_controls=False`, to make the intent clear.